lib/symboldatabase.cpp

```diff
@@ -2819,12 +2819,29 @@ void SymbolDatabase::printXml(std::ostream &out) const
         out << " isPointer=\""      << var->isPointer() << '\"';
         out << " isReference=\""    << var->isReference() << '\"';
         out << " isStatic=\""       << var->isStatic() << '\"';
+        out << " access=\""         << access_control_to_string(var->_access) << '\"';
         out << "/>" << std::endl;
     }
     out << "  </variables>" << std::endl;
     out << std::resetiosflags(std::ios::boolalpha);
 }
 
+std::string SymbolDatabase::access_control_to_string(const AccessControl& access) const {
+    switch (access) {
+    case Public:      return "Public";
+    case Protected:   return "Protected";
+    case Private:     return "Private";
+    case Global:      return "Global";
+    case Namespace:   return "Namespace";
+    case Argument:    return "Argument";
+    case Local:       return "Local";
+    case Throw:       return "Throw";
+    default:
```

Comment:
This "default:" means that gcc won't warn if there is a missing enum constant like:

    enum1.c:5:3: warning: enumeration value ‘C’ not handled in switch [-Wswitch]

So please remove it.